### ISSUE TITLE
[Bug] software emulation is enabled but still failed with /dev/kvm not found

### ISSUE THREAD

**Issue by guangbochen** [kubevirt/kubevirt#6218]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
software emulation is enabled for my testing kind cluster but after starting a new VM it still failed with /dev/kvm not found error:
```
  Normal   SuccessfulCreate  6m2s                  virtualmachine-controller  Created virtual machine pod virt-launcher-test-ngcfg-7cnmb
  Warning  SyncFailed        67s (x15 over 2m30s)  virt-handler               stat /proc/21406/root/dev/kvm: no such file or directory
```

**What you expected to happen**:
the `/dev/kvm` should not be required when the software emulation is enabled

**How to reproduce it (as minimally and precisely as possible)**:
- deploy the kind cluster with set kubevirt CR to 
```YAML
spec:
  configuration:
    developerConfiguration:
      useEmulation: true
```
- deploy a simple VM and waiting for the to get up and running
- VM is never ready because of the `stat /proc/21406/root/dev/kvm` error

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`): v0.44.0
- Kubernetes version (use `kubectl version`): v1.12.1
- VM or VMI specifications: 
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by xpivarc** [comment]
This will be fixed by https://github.com/kubevirt/kubevirt/pull/6190

**Comment by zcahana** [comment]
#6190 has been merged to `main`, also backported to `v0.44.1` via #6236.
Thanks for reporting @guangbochen!

/close
